play_book_pdf_tool/play_book_pdf_tool.py

```diff
@@ -59,7 +59,9 @@ def create_pdf(image_paths, output_pdf_path):
             im.read(1)
 
     output_pdf = open(output_pdf_path, "wb")
-    img2pdf.convert(*image_paths, outputstream=output_pdf)
+    a4inpt = (img2pdf.mm_to_pt(210),img2pdf.mm_to_pt(297))
```

Comment:
Why always A4? Could the actual page size be inferred from the metadata?

Reply:
I used A4 as the first thing that came to mind, having checked that the proportion of the page images is about the same as in it (1:√2).

But there was a thought of extracting the page size information from somewhere, or taking it from the page images.....

Reply:
I used A4 as the first thing that came to mind, having checked that the proportion of the page images is about the same as in it (1:√2).

But there was a thought of extracting the page size information from somewhere, or taking it from the page images...

Reply:
I don't know how to get the page size but you can definitely not assume that it's A4 unfortunately…